Entity/FormSubmission.php

```diff
@@ -187,7 +187,7 @@ public function getLabel(): string
         return $this->label;
     }
 
-    public function getDeadlineDate(): string
+    public function getDeadlineDate(): ?string
     {
```

Comment:
Why does a date is in a string type?

Reply:
Nice catch, I overlooked this earlier!

Reply:
OK, this was to allow any date format, but indeed not a good idea.